tests/unit/generateSystemNumberFunction.ts

```diff
@@ -0,0 +1,45 @@
+import { generateSystemNumber } from "../../src/functions/generateSystemNumber";
+import { NumberService } from "../../src/services/NumberService";
+import mockContext from "aws-lambda-mock-context";
+import {HTTPResponse} from "../../src/utils/HTTPResponse";
+
+describe("generate SystemNumber Function", () => {
+    it("should invoke NumberService", async () => {
+        let ctx: any = mockContext();
```

Comment:
mockContext has proven more trouble  than it's worth. Suggest replacing universally with
`@ts-ignore
const ctx: Context = null`
where Context is from the aws-lambda library